components/schema-migrator/migrations/director/20240328144434_update-at-and-created-at-on-tenants.up.sql

```diff
@@ -0,0 +1,48 @@
+BEGIN;
+
+-- Add columns
+ALTER TABLE business_tenant_mappings
+    ADD COLUMN created_at timestamp,
```

Comment:
You can use `DEFAULT` value during column creation instead of updating each row of the table below.

Something like this:
`ADD COLUMN created_at TIMESTAMP DEFAULT NOW()`

Then the two update statements below can be deleted.